src/GraphQL/Types/Relay/ConnectionType.cs

```diff
@@ -1,9 +1,19 @@
 namespace GraphQL.Types.Relay
 {
+    /// <summary>
+    /// A connection graph type for the specified node graph type. The GraphQL type name
+    /// defaults to {NodeType}Connection where {NodeType} is the GraphQL type name of
+    /// the node graph type. This graph type assumes that the source (the result of
+    /// the parent field's resolver) is <see cref="ConnectionType{TNodeType, TEdgeType}"/>
+    /// or <see cref="ConnectionType{TNodeType}"/> or has the same members.
+    /// </summary>
+    /// <typeparam name="TNodeType">The graph type of the result data set's data type.</typeparam>
+    /// <typeparam name="TEdgeType">The edge graph type of node, typically <see cref="EdgeType{TNodeType}"/>.</typeparam>
     public class ConnectionType<TNodeType, TEdgeType> : ObjectGraphType<object>
         where TNodeType : IGraphType
         where TEdgeType : EdgeType<TNodeType>
     {
+        /// <inheritdoc/>
```

Comment:
I think that description for `edges` field is wrong, it seems to be copy-pasted from `pageInfo` field description.